src/main/java/org/openshift/jenkins/plugins/openshiftlogin/OpenShiftOAuth2SecurityRealm.java

```diff
@@ -1154,7 +1154,10 @@ public void doLogout(StaplerRequest req, StaplerResponse resp) throws IOExceptio
         if (idTokenResponse != null) {
             LOGGER.info("Found an  oauthaccess token in the sessions");
             String oAuthToken = idTokenResponse.getAccessToken();
-            String oAuthtokenName = tokenToObjectName(oAuthToken);
+            String oAuthtokenName = oAuthToken;
```

Comment:
Hi @barleyer , thanks for reporting this and submitting the PR. 
regarding this code portion, it is preferable to compute the `oAuthtokenName` consistently in the  `tokenToObjectName ` method and thus to add this test in this method.

based on your tests, and your comments, can you confirm that for 4.5- ,  idTokenResponse.getAccessToken() is the actual token name?

On our side, we haven't released the 4.5 image that tests this.

Reply:
Yeah,the value of token is the object name of the token.